src/modules-load/modules-load.c

```diff
@@ -39,12 +45,25 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
         return 0;
 }
 
-static int apply_file(struct kmod_ctx *ctx, const char *path, bool ignore_enoent) {
+static int send_module_to_load(const int sock, const char *module) {
```

Comment:
"const int sock" doesn't really make sense. use const for pointers, not for local variables please. jsut drop the "const".

Reply:
Reworked.